tests/components/fritz/test_image.py

```diff
@@ -24,6 +24,7 @@
 GUEST_WIFI_ENABLED: dict[str, dict] = {
     "WLANConfiguration0": {},
     "WLANConfiguration1": {
+        "GetBeaconAdvertisement": {"NewBeaconAdvertisementEnabled": 1},
```

Comment:
these properties are now checked in the lib for creating the wifi connect qr codes. they were not needed before, so they were not present..
I've also duple checked it with my fritzbox at it still works :+1: